echidna/scripts/dump_shrink.py

```diff
@@ -0,0 +1,100 @@
+""" Example shrinking script
+
+This script:
+  * Reads in spectrum from hdf5
+  * shrinks spectrum with user input pars and low high values
+  * Saves shrunk spectrum to given output or, by default, to the same
+    directory with _shrink added to the filename.
+
+Examples:
+  To shrink hdf5 file ``example.hdf5`` that has dimensions energy_mc and
+  radial3_mc to a range in energy of 1. to 4. MeV and radial3 from 0. to 0.5
```

Comment:
`energy_mc` and `radial3_mc` may benefit from the inline-code formatting, in the same way you formatted `example.hdf5`